k8s/eks/README.md

```diff
@@ -0,0 +1,284 @@
+This guide will help you setup an EKS (Elastic Kubernetes Service) cluster, and run testground plans on it.
+
+## Requirements
+
+It is assumed that you already have the following:
+
+- A bastion host on AWS, or a laptop with AWS credentials which will be used to run the installation script, interact with the cluster, and run testground plans
+- The laptop or bastion host has the following software installed:
+
+**Note: the software and installation guides have been tested on amazon linux. Please verify which OS are you using and follow the official guides accordingly.
+Links have been included for each software requirement. Listed guides are for example purposes only.**
+
+**1. docker**
+
+Official installation guide:
+
+https://docs.docker.com/engine/install/ubuntu/
+
+Installation on amazon linux:
+
+```
+sudo yum install docker -y
+sudo systemctl enable docker.service
+sudo systemctl start docker.service
+sudo usermod -aG docker ec2-user
+newgrp docker
+```
+
+**2. helm**
+
+Official installation guide:
+
+https://helm.sh/docs/intro/install/
+
+```
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
```

Comment:
We could of course, and some some third-party dependencies we have included this as well. 
But have in mind that this needs to be tested and updated frequently, as it is easy to reach the software rot stage.

Reply:
In the current form, this will become untested as soon as https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 starts returning a new version.

Please use pinned versions where possible.

Reply:
Ok, we will use pinned versions where possible.

Reply:
Specific versions of software used during writing of this have been added to the README where applicable.